packages/console/src/components/RoleUsersTransfer/components/SourceUserItem/index.tsx

```diff
@@ -36,8 +33,8 @@ function SourceUserItem({ user, isSelected, onSelect }: Props) {
           onSelect();
         }}
       />
-      <UserAvatar user={user} size="micro" />
-      <div className={styles.title}>{getUserTitle(user) ?? t('users.unnamed')}</div>
+      <UserAvatar hasTooltip user={user} size="micro" />
```

Comment:
nit: do you think implement a ToolTip wrapper could be a better approach?  

```jsx
<WithUserTip>
  <UserAvatar />
</WithUserTip>
```

Reply:
That's also doable, but personally I don't have a preference on which one is better than the other one.

Reply:
I've been thinking about this and it's actually a bit cumbersome than it sounds like. Also there's no other component can use this `<WithUserTip />` component atm. So I decided to save the effort and go ahead without this change.